src/maps/OverlayConfig.h

```diff
@@ -18,11 +18,16 @@
 #ifndef SRC_MAPS_OVERLAY_CONFIG_H_
 #define SRC_MAPS_OVERLAY_CONFIG_H_
 
+#include <string>
+
 namespace maps {
 
 struct OverlayConfig {
     bool drawMyAircraft = true;
     bool drawOtherAircraft = true;
+    std::string colorOtherAircraftBelow;
```

Comment:
Would it be possible to use uints here and do the name to uint resolution one layer higher? That way, I could integrate color changing at runtime. The color picker widget returns uints, so with the current design I'd have to convert them into strings and it would be nice if that extra step could be avoided.

Reply:
I've moved this into the `Settings` class, and made some other enhancements.
It would be good to add UI support for editing the color. I didn't know there was a color picker widget. I may take a look at this, although probably not in the very near future.